templates/2025/navbar.html

```diff
@@ -87,11 +89,11 @@
                   <a class="nav-link g-px-0 g-color-black" href="/2025/faqs/">FAQs</a>
                 </li> 
                 -->
-                <li class="dropdown-item">
+                <!-- <li class="dropdown-item">
                   <a class="nav-link g-px-0 g-color-black" href="/2025/h&g/"
                     >Health and Safety Guidelines</a
```

Comment:
We could as well delete this(the comment plus the list items) as is dead code if is catered for under Attendee Guide